### ISSUE TITLE
Broken dependencies on master 

### ISSUE THREAD

**Issue by shivam-Purohit** [litmuschaos/litmusctl#186]
**Description**
When trying to build binary for master branch using command `go build`. It throws this error
```
../../../go/pkg/mod/github.com/argoproj/argo-workflows/v3@v3.3.1/pkg/apis/workflow/common.go:6:2: missing go.sum entry for module providing package k8s.io/client-go/rest (imported by github.com/litmuschaos/litmusctl/pkg/k8s); to add:

	go get github.com/litmuschaos/litmusctl/pkg/k8s

../../../go/pkg/mod/github.com/argoproj/argo-workflows/v3@v3.3.1/pkg/apis/workflow/common.go:7:2: missing go.sum entry for module providing package k8s.io/client-go/tools/clientcmd/api (imported by github.com/argoproj/argo-workflows/v3/pkg/apis/workflow); to add:

	go get github.com/argoproj/argo-workflows/v3/pkg/apis/workflow@v3.3.1

../../../go/pkg/mod/github.com/argoproj/argo-workflows/v3@v3.3.1/pkg/apis/workflow/v1alpha1/container_set_template_types.go:7:2: missing go.sum entry for module providing package k8s.io/api/core/v1 (imported by github.com/litmuschaos/litmusctl/pkg/k8s); to add:

	go get github.com/litmuschaos/litmusctl/pkg/k8s

../../../go/pkg/mod/github.com/argoproj/argo-workflows/v3@v3.3.1/pkg/apis/workflow/v1alpha1/generated.pb.go:16:2: missing go.sum entry for module providing package k8s.io/api/policy/v1beta1 (imported by github.com/argoproj/argo-workflows/v3/pkg/apis/workflow/v1alpha1); to add:

	go get github.com/argoproj/argo-workflows/v3/pkg/apis/workflow/v1alpha1@v3.3.1

../../../go/pkg/mod/github.com/argoproj/argo-workflows/v3@v3.3.1/pkg/apis/workflow/v1alpha1/workflow_types.go:19:2: missing go.sum entry for module providing package k8s.io/apimachinery/pkg/api/resource (imported by github.com/argoproj/argo-workflows/v3/pkg/apis/workflow/v1alpha1); to add:

	go get github.com/argoproj/argo-workflows/v3/pkg/apis/workflow/v1alpha1@v3.3.1

../../../go/pkg/mod/github.com/argoproj/argo-workflows/v3@v3.3.1/pkg/apis/workflow/v1alpha1/cluster_workflow_template_types.go:6:2: missing go.sum entry for module providing package k8s.io/apimachinery/pkg/apis/meta/v1 (imported by github.com/litmuschaos/litmusctl/pkg/infra_ops); to add:

	go get github.com/litmuschaos/litmusctl/pkg/infra_ops

../../../go/pkg/mod/github.com/argoproj/argo-workflows/v3@v3.3.1/pkg/apis/workflow/v1alpha1/workflow_types.go:21:2: missing go.sum entry for module providing package k8s.io/apimachinery/pkg/labels (imported by github.com/argoproj/argo-workflows/v3/pkg/apis/workflow/v1alpha1); to add:

	go get github.com/argoproj/argo-workflows/v3/pkg/apis/workflow/v1alpha1@v3.3.1

../../../go/pkg/mod/github.com/argoproj/argo-workflows/v3@v3.3.1/pkg/apis/workflow/v1alpha1/register.go:5:2: missing go.sum entry for module providing package k8s.io/apimachinery/pkg/runtime (imported by github.com/argoproj/argo-workflows/v3/pkg/apis/workflow/v1alpha1); to add:

	go get github.com/argoproj/argo-workflows/v3/pkg/apis/workflow/v1alpha1@v3.3.1

../../../go/pkg/mod/github.com/argoproj/argo-workflows/v3@v3.3.1/pkg/apis/workflow/v1alpha1/common.go:5:2: missing go.sum entry for module providing package k8s.io/apimachinery/pkg/runtime/schema (imported by github.com/litmuschaos/litmusctl/pkg/k8s); to add:

	go get github.com/litmuschaos/litmusctl/pkg/k8s

../../../go/pkg/mod/github.com/argoproj/argo-workflows/v3@v3.3.1/pkg/apis/workflow/v1alpha1/cron_workflow_types.go:6:2: missing go.sum entry for module providing package k8s.io/apimachinery/pkg/types (imported by github.com/argoproj/argo-workflows/v3/pkg/apis/workflow/v1alpha1); to add:

	go get github.com/argoproj/argo-workflows/v3/pkg/apis/workflow/v1alpha1@v3.3.1

../../../go/pkg/mod/github.com/argoproj/argo-workflows/v3@v3.3.1/pkg/apis/workflow/v1alpha1/container_set_template_types.go:8:2: missing go.sum entry for module providing package k8s.io/apimachinery/pkg/util/intstr (imported by github.com/argoproj/argo-workflows/v3/pkg/apis/workflow/v1alpha1); to add:

	go get github.com/argoproj/argo-workflows/v3/pkg/apis/workflow/v1alpha1@v3.3.1

../../../go/pkg/mod/github.com/argoproj/argo-workflows/v3@v3.3.1/pkg/apis/workflow/v1alpha1/validation_utils.go:9:2: missing go.sum entry for module providing package k8s.io/apimachinery/pkg/util/validation (imported by github.com/argoproj/argo-workflows/v3/pkg/apis/workflow/v1alpha1); to add:

	go get github.com/argoproj/argo-workflows/v3/pkg/apis/workflow/v1alpha1@v3.3.1

../../../go/pkg/mod/github.com/argoproj/argo-workflows/v3@v3.3.1/pkg/apis/workflow/v1alpha1/container_set_template_types.go:9:2: missing go.sum entry for module providing package k8s.io/apimachinery/pkg/util/wait (imported by github.com/argoproj/argo-workflows/v3/pkg/apis/workflow/v1alpha1); to add:

	go get github.com/argoproj/argo-workflows/v3/pkg/apis/workflow/v1alpha1@v3.3.1

../../../go/pkg/mod/k8s.io/kube-openapi@v0.0.0-20220124234850-424119656bbf/pkg/common/common.go:23:2: github.com/go-openapi/jsonpointer@v0.19.6: missing go.sum entry for go.mod file; to add it:

	go mod download github.com/go-openapi/jsonpointer

../../../go/pkg/mod/k8s.io/kube-openapi@v0.0.0-20220124234850-424119656bbf/pkg/validation/spec/ref.go:23:2: missing go.sum entry for module providing package github.com/go-openapi/jsonreference (imported by k8s.io/kube-openapi/pkg/validation/spec); to add:

	go get k8s.io/kube-openapi/pkg/validation/spec@v0.0.0-20220124234850-424119656bbf

../../../go/pkg/mod/k8s.io/kube-openapi@v0.0.0-20220124234850-424119656bbf/pkg/validation/spec/header.go:20:2: missing go.sum entry for module providing package github.com/go-openapi/swag (imported by k8s.io/kube-openapi/pkg/validation/spec); to add:

	go get k8s.io/kube-openapi/pkg/validation/spec@v0.0.0-20220124234850-424119656bbf

../../../go/pkg/mod/github.com/litmuschaos/chaos-operator@v0.0.0-20230718113617-6819a4be12e4/api/litmuschaos/v1alpha1/chaosexperiment_types.go:21:2: missing go.sum entry for module providing package k8s.io/api/rbac/v1 (imported by github.com/litmuschaos/chaos-operator/api/litmuschaos/v1alpha1); to add:

	go get github.com/litmuschaos/chaos-operator/api/litmuschaos/v1alpha1@v0.0.0-20230718113617-6819a4be12e4

pkg/k8s/operations.go:41:2: missing go.sum entry for module providing package k8s.io/api/authorization/v1 (imported by github.com/litmuschaos/litmusctl/pkg/k8s); to add:

	go get github.com/litmuschaos/litmusctl/pkg/k8s

pkg/k8s/operations.go:43:2: missing go.sum entry for module providing package k8s.io/apimachinery/pkg/api/errors (imported by github.com/litmuschaos/litmusctl/pkg/k8s); to add:

	go get github.com/litmuschaos/litmusctl/pkg/k8s

pkg/k8s/operations.go:44:2: missing go.sum entry for module providing package k8s.io/apimachinery/pkg/api/meta (imported by github.com/litmuschaos/litmusctl/pkg/k8s); to add:

	go get github.com/litmuschaos/litmusctl/pkg/k8s

pkg/k8s/operations.go:46:2: missing go.sum entry for module providing package k8s.io/apimachinery/pkg/apis/meta/v1/unstructured (imported by github.com/litmuschaos/litmusctl/pkg/k8s); to add:

	go get github.com/litmuschaos/litmusctl/pkg/k8s

pkg/k8s/operations.go:48:2: missing go.sum entry for module providing package k8s.io/apimachinery/pkg/util/yaml (imported by github.com/litmuschaos/litmusctl/pkg/k8s); to add:

	go get github.com/litmuschaos/litmusctl/pkg/k8s

pkg/k8s/operations.go:49:2: missing go.sum entry for module providing package k8s.io/client-go/discovery (imported by github.com/litmuschaos/litmusctl/pkg/k8s); to add:

	go get github.com/litmuschaos/litmusctl/pkg/k8s

pkg/k8s/operations.go:31:2: missing go.sum entry for module providing package k8s.io/client-go/discovery/cached/memory (imported by github.com/litmuschaos/litmusctl/pkg/k8s); to add:

	go get github.com/litmuschaos/litmusctl/pkg/k8s

pkg/k8s/operations.go:32:2: missing go.sum entry for module providing package k8s.io/client-go/dynamic (imported by github.com/litmuschaos/litmusctl/pkg/k8s); to add:

	go get github.com/litmuschaos/litmusctl/pkg/k8s

pkg/k8s/client.go:24:2: missing go.sum entry for module providing package k8s.io/client-go/kubernetes (imported by github.com/litmuschaos/litmusctl/pkg/k8s); to add:

	go get github.com/litmuschaos/litmusctl/pkg/k8s

pkg/k8s/operations.go:50:2: missing go.sum entry for module providing package k8s.io/client-go/kubernetes/typed/authorization/v1 (imported by github.com/litmuschaos/litmusctl/pkg/k8s); to add:

	go get github.com/litmuschaos/litmusctl/pkg/k8s

pkg/k8s/client.go:25:2: missing go.sum entry for module providing package k8s.io/client-go/plugin/pkg/client/auth (imported by github.com/litmuschaos/litmusctl/pkg/k8s); to add:

	go get github.com/litmuschaos/litmusctl/pkg/k8s

pkg/k8s/operations.go:35:2: missing go.sum entry for module providing package k8s.io/client-go/restmapper (imported by github.com/litmuschaos/litmusctl/pkg/k8s); to add:

	go get github.com/litmuschaos/litmusctl/pkg/k8s

pkg/k8s/client.go:26:2: missing go.sum entry for module providing package k8s.io/client-go/tools/clientcmd (imported by github.com/litmuschaos/litmusctl/pkg/k8s); to add:

	go get github.com/litmuschaos/litmusctl/pkg/k8s

pkg/k8s/client.go:27:2: missing go.sum entry for module providing package k8s.io/client-go/util/homedir (imported by github.com/litmuschaos/litmusctl/pkg/k8s); to add:

	go get github.com/litmuschaos/litmusctl/pkg/k8s

```

Fix : 
`go get ` to get the missing dependencies.

**Comment by SarthakJain26** [comment]
@shivam-Purohit Is this issue still valid? I think it was something with your dev setup

**Comment by shivam-Purohit** [comment]
@SarthakJain26 I think this is fixed in commit [8d346af71c82163b453e6c08785ee025a54b9abf](https://github.com/litmuschaos/litmusctl/pull/191/commits).
